Mathlib/RingTheory/Nilpotent/Basic.lean

```diff
@@ -3,14 +3,10 @@ Copyright (c) 2021 Oliver Nash. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Oliver Nash
 -/
-import Mathlib.Algebra.Algebra.Bilinear
 import Mathlib.Algebra.GeomSum
-import Mathlib.Algebra.GroupPower.Ring
-import Mathlib.Algebra.GroupWithZero.NonZeroDivisors
-import Mathlib.Data.Nat.Choose.Sum
-import Mathlib.LinearAlgebra.Matrix.ToLin
-import Mathlib.LinearAlgebra.Quotient
-import Mathlib.RingTheory.Ideal.Operations
+import Mathlib.Algebra.Module.Basic
+import Mathlib.Algebra.SMulWithZero
+import Mathlib.Data.Nat.Lattice
```

Comment:
With this set of imports, it might arguably even make sense to move this file to `Algebra.GroupWithZero`. I'll leave that to your judgement though: it also makes sense to keep all the files on nilpotent elements together in one folder.

Reply:
I have no strong feeling on this.